src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -170,12 +170,22 @@ public function getSubresourceIriFromResourceClass(string $resourceClass, array
     /**
      * Generate the identifier url.
      *
-     * @param array $identifiers
+     * @param array  $identifiers
+     * @param object $resourceClass
```

Comment:
`$resourceClass` is a `string`. You should also add the type hint for the parameter.